stream-configuration/src/main/java/com/backbase/stream/config/BackbaseStreamConfigurationProperties.java

```diff
@@ -53,6 +53,10 @@ public static class DbsConnectionProperties {
          */
         private String approvalsBaseUrl = "http://approval-service:8080";
 
+        /**
+         * The location of DBS User Profile Manager Service.
+         */
+        private String paymentOrderBaseUrl = "http://payment-order-service/:8080";
```

Comment:
there is a typo there with the slash before the colon

Reply:
Good catch! Fixed.